ui/webpack.prod.js

```diff
@@ -21,7 +21,7 @@ module.exports = merge(common, {
   output: {
     filename: 'bundle.js',
     path: path.resolve(__dirname, 'dist'),
-    publicPath: '/assets/',
+    publicPath: 'auto',
```

Comment:
Allow [webpack to auto figure out the path](https://webpack.js.org/guides/public-path/#automatic-publicpath) for all other assets based on the the location of the entry asset served

Reply:
Oh I didn't know that. Looking better, thanks!